src/derandomize.rs

```diff
@@ -0,0 +1,321 @@
+// sablast: Spectral Burrows-Wheeler transform accelerated local alignment search
+//
+// Copyright 2024 Tommi Mäklin [tommi@maklin.fi].
+
+// Copyrights in this project are retained by contributors. No copyright assignment
+// is required to contribute to this project.
+
+// Except as otherwise noted (below and/or in individual files), this
+// project is licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE> or <http://www.apache.org/licenses/LICENSE-2.0> or
+// the MIT license, <LICENSE-MIT> or <http://opensource.org/licenses/MIT>,
+// at your option.
+//
+//! Derandomizing noisy _k_-bounded matching statistics.
+
+/// Evaluates the CDF of _k_-bounded matching statistics random match distribution.
+///
+/// Computes the log-probability that a matching statistic with value
+/// `t` or less that is the result of mapping a _k_-mer with
+/// `alphabet_size` possible characters against an index containing
+/// `n_kmers` _k_-mers was generated by chance.
+///
+/// TODO Add formula to log_rm_max_cdf documentation
```

Comment:
A new Todo was discovered. If it is not a priority right now,consider marking it for later attention.
TODO Add formula to log_rm_max_cdf documentation
- [ ] Ignore